versions.props

```diff
@@ -3,42 +3,42 @@ com.atlassian.commonmark:* = 0.12.1
 com.fasterxml.jackson.core:jackson-databind = 2.14.0
 com.google.code.findbugs:jsr305 = 3.0.2
 com.google.guava:guava = 31.1-jre
-com.palantir.common:streams = 2.0.0
-com.palantir.conjure.java.api:* = 2.28.0
-com.palantir.conjure.java.runtime:* = 7.38.0
+com.palantir.common:streams = 2.1.0
+com.palantir.conjure.java.api:* = 2.29.0
+com.palantir.conjure.java.runtime:* = 7.39.0
 com.palantir.conjure.verification:* = 0.19.0
 com.palantir.conjure:* = 4.28.0
-com.palantir.dialogue:* = 3.69.0
+com.palantir.dialogue:* = 3.71.0
 com.palantir.ri:resource-identifier = 2.5.0
-com.palantir.safe-logging:* = 3.1.0
+com.palantir.safe-logging:* = 3.2.0
 com.palantir.syntactic-paths:syntactic-paths = 0.9.0
 com.palantir.tokens:* = 3.16.0
 com.palantir.tracing:* = 6.15.0
 com.palantir.websecurity:dropwizard-web-security = 1.1.0
 com.squareup.okhttp3:* = 3.14.1
 com.squareup:javapoet = 1.13.0
-info.picocli:picocli = 4.6.3
+info.picocli:picocli = 4.7.0
 io.dropwizard.metrics:metrics-core = 4.1.0
 io.dropwizard:dropwizard-* = 2.0.34
 io.undertow:undertow-core = 2.2.19.Final
 jakarta.annotation:jakarta.annotation-api = 1.3.5
 jakarta.validation:jakarta.validation-api = 2.0.2
 jakarta.ws.rs:jakarta.ws.rs-api = 2.1.6
-joda-time:joda-time = 2.11.1
+joda-time:joda-time = 2.12.1
 org.apache.commons:commons-lang3 = 3.12.0
 org.assertj:* = 3.23.1
 org.glassfish.jersey.*:* = 2.34
 org.hamcrest:hamcrest-core = 2.2
 org.immutables:value = 2.8.8
 org.javassist:javassist = 3.22.0-GA
 org.jetbrains:annotations = 23.0.0
-org.junit.jupiter:* = 5.8.2
-org.junit.vintage:* = 5.8.2
-org.mockito:* = 4.6.1
+org.junit.jupiter:* = 5.9.1
+org.junit.vintage:* = 5.9.1
```

Comment:
this bump is causing [build failures](https://app.circleci.com/pipelines/github/palantir/conjure-java/3727/workflows/e4a87c6e-870a-4c35-a86e-25873ee2e825/jobs/23466?invite=true#step-106-200) similar to dialogue, see need something like https://github.com/palantir/dialogue/pull/1815 to migrate to JUnit 5 fully and off vintage.